cfpb/sbl-frontend
# feat(salesforce): add placeholder page with links to the designs

Adds a quick placeholder page for "Update your financial institution profile" and "Complete your user profile" so stakeholders wonder why we're using a generic SBL Help form for these pages. The placeholder includes a link to the future design image. Something that @natalia-fitzgerald requested, and I think makes sense to avoid weird stakeholder questions. Will be removed after the proof of concept is done next sprint, see: #214 

Closes #213 

## Changes
- makes all links to SBL Help / Salesforce go through `common.ts` so that we can centralize everything and also start linking out to the three different forms for future work next sprint:
  - SBL Help (generic)
  - Update your financial institution profile
  - Complete your user profile
- adds a placeholder Salesforce page so that stakeholders understand why there isn't a form here
- links to designs for future Salesforce forms

## How to test this PR

1. When I click on "Update your financial institution profile" and "Complete your user profile" links, do I see an explanatory page with a link to the design?

## Screenshots

![Screenshot 2024-01-31 at 1 07 48 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/af225e73-f9d6-46d4-a21c-59ab92306a0c)
![Screenshot 2024-01-31 at 1 08 17 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/b655114f-ea91-4d0e-94b8-526b69c04b1b)
